tests/cases/providers_care_platform/case_providers_home.yaml

```diff
@@ -497,4 +497,102 @@ TestProviderHelpMenuOption:
           Var: HELP_PAGE_URL
           Value: .pdf
 
- 
\ No newline at end of file
+# CXTB-185 Provider Can Click and View Facility Device Record from Listing
+TestProviderCanClickAndViewFacilitiesDevice:
+  Environment: Settings
+  Roles:
+    - Role: desktopChrome
+      App: "desktop"
+  Actions:
+    # Login-in provider Care Partner
+    - Type: case
+      Value: ProvidersCarePlatformLogin
+    # Click facilities button
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.facilities_button$
+    # click on the first listing record
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.first_senior_lastname$
+    # Click on Overview button and verify if tab showing belongs to Overview
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.overview_button$
+    - Type: get_attribute
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.tabs_content$
+      Greps:
+      - var: TAB_CLASS
+        attr: tab-title
+        condition: nempty
+        match: "(.*)"
+    - Type: assert
+      Asserts:
+      - Type: contain
+        Var: TAB_CLASS
+        Value: 'Overview'
+    # Click on Local Hospital button and verify if tab showing belongs to Local Hospital
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.local_hospitals_button$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.title_subtab_local_hospital$
+    - Type: get_attribute
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.tabs_content$
+      Greps:
+      - var: TAB_TITLE
+        attr: tab-title
+        condition: nempty
+        match: "(.*)"
+    - Type: assert
+      Asserts:
+        - Type: contain
+          Var: TAB_TITLE
+          Value: 'Local Hospitals'
+    # Click on Devices button and verify if tab showing belongs to Devices
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.devices_button$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.title_subtab_devices$
+    - Type: get_attribute
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.tabs_content$
+      Greps:
+        - var: TAB_TITLE
+          attr: tab-title
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Asserts:
+        - Type: contain
+          Var: TAB_TITLE
+          Value: 'Devices'
+    # Click on Record button and check the Devices
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.records_button$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility_Record.info_title$
+    # Click to check Devices Tab
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility_Record.devices_button$
```

Comment:
This is a care_platform page it should be  provider_care_platform_facility_record